src/components/Modal/AddInternForm.js

```diff
@@ -3,65 +3,84 @@ import {
                 { name: 'DIIT', id: 15 },
                 { name: 'Exchng Prog.', id: 16 },
                 { name: 'MDes', id: 17 },
-                { name: 'Msc-PhD(Dual)', id: 18 },
+                { name: 'MSc-PhD(Dual)', id: 18 },
                 { name: 'MTech-PhD', id: 19 },
+                { name: 'MS-Research', id: 20 },
+                { name: 'PGPEX-VLM', id: 21 },
             ],
 
             departments: [
-                { name: 'Open to ALL', id: 1 },
-                { name: 'Aerospace', id: 2 },
-                { name: 'BSBE', id: 3 },
-                { name: 'Chemical', id: 4 },
-                { name: 'Chemistry', id: 5 },
-                { name: 'Civil', id: 6 },
-                { name: 'Computer Science', id: 7 },
-                { name: 'Earth Science', id: 8 },
+                { name: 'Aerospace Engg.', id: 1 },
+                { name: 'Biol.Sci. And Bio.Engg.', id: 2 },
+                { name: 'Chemical Engg.', id: 3 },
+                { name: 'Chemistry', id: 4 },
+                { name: 'Civil Engg.', id: 5 },
+                { name: 'Cognitive Sciences', id: 6 },
+                { name: 'Computer Science & Engg.', id: 7 },
+                { name: 'Earth Sciences', id: 8 },
                 { name: 'Economics', id: 9 },
-                { name: 'Electrical', id: 10 },
-                { name: 'Material Science', id: 11 },
-                { name: 'Mathematics', id: 12 },
-                { name: 'Mechanical', id: 13 },
-                { name: 'Physics', id: 14 },
-                { name: 'Cognitive Science', id: 15 },
-                { name: 'Material Science', id: 16 },
-                { name: 'Statistics', id: 17 },
-                { name: 'Photonics Science & Enginee.', id: 18 },
+                { name: 'Electrical Engg.', id: 10 },
+                { name: 'Humanities & Soc. Sciences', id: 11 },
+                { name: 'Ind. & Management Engg.', id: 12 },
+                { name: 'Master Of Design', id: 13 },
+                { name: 'Materials Science & Engg.', id: 14 },
+                { name: 'Materials Science Programme', id: 15 },
+                { name: 'Mathematics', id: 16 },
+                { name: 'Math For Pg Online', id: 17 },
+                { name: 'Mechanical Engineering', id: 18 },
                 { name: 'Nuclear Eng. and Tech. Prog.', id: 19 },
-                { name: 'Statistics', id: 20 },
+                { name: 'Photonics Science & Engg.', id: 20 },
+                { name: 'Physics', id: 21 },
+                { name: 'Statistics', id: 22 },
+            ],
+
+            batches: [
+                { name: 'Y20', id: 1 },
+                { name: 'Y19', id: 2 },
+                { name: 'Y18', id: 3 },
+                { name: 'Y17', id: 4 },
+                { name: 'Y16', id: 5 },
+                { name: 'Y15', id: 6 },
+                { name: 'Y14', id: 7 },
+                { name: 'Y13', id: 8 },
             ],
```

Comment:
using these 2 times in different files. advisable to create a different file as constant and import them here.

Reply:
I have created a single form instead of two.